.github/workflows/publish_to_pypi.yml

```diff
@@ -0,0 +1,49 @@
+name: Publish xrpl-py 🐍 distribution 📦 to PyPI
+env:
+  POETRY_VERSION: 1.8.3
+
+on: push
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.x"
+    - name: Load cached .local
+      id: cache-poetry
+      uses: actions/cache@v3
+      with:
+          path: /home/runner/.local
+          key: dotlocal-${{ env.POETRY_VERSION }}
+    - name: Install poetry
+      if: steps.cache-poetry.outputs.cache-hit != 'true'
+      run: |
+        curl -sSL https://install.python-poetry.org/ | python - --version ${{ env.POETRY_VERSION }}
+        echo "$HOME/.local/bin" >> $GITHUB_PATH
+    - name: Build a binary wheel and a source tarball
+      run: poetry build
+    - name: Store the distribution packages
+      uses: actions/upload-artifact@v4
+      with:
+        name: python-package-distributions
+        path: dist/
+  publish-to-pypi:
+    name: >-
+        Publish Python 🐍 distribution 📦 to PyPI
+    if: startsWith(github.ref, 'refs/tags/')  # only publish to PyPI on tag pushes
+    runs-on: ubuntu-latest
+    permissions:
+        id-token: write  # IMPORTANT: mandatory for trusted publishing
```

Comment:
Link to docs

Reply:
are you referring to Github Actions docs?
`id-token` refers to the temporary OpenConnectID token that's assigned to Github Actions by PyPI. I've included docs inside the `CONTRIBUTING.md` file.

Reply:
When you say something is mandatory, it’s best to link to why

Reply:
fixed in [71f529e](https://github.com/XRPLF/xrpl-py/pull/760/commits/71f529e9904306af87e4bce0ff8ffd40e68e658b)